Document/0x06b-Basic-Security-Testing.md

```diff
@@ -27,36 +27,63 @@ If you want to get serious with iOS security testing, you need a Mac, for the si
+
+#### Types of Jailbreaking Methods
 
 In jailbreak lingo, we talk about tethered and untethered jailbreaking methods. In the "tethered" scenario, the jailbreak doesn't persist throughout reboots, so the device must be connected (tethered) to a computer during every reboot to re-apply it. "Untethered" jailbreaks need only be applied once, making them the most popular choice for end users.
 
-Jailbreaking methods vary across iOS versions. Best choice is to [check if a public jailbreak is available for your iOS version](https://canijailbreak.com/ "Can I Jailbreak? by IPSW Downloads"). Beware of fake tools and spyware that is often distributed around the Internet, often hiding behind domain names similar to the jailbreaking group/author.
+#### Benefits of Jailbreaking
+While a standard user will want to jailbreak in order to tweak the iOS system appearance, add new features or install _free_ third party apps. The benefits of jailbreaking an iOS Device for a security tester include but are not limited to the following:
+
+- Removing the part of the security (and other) limitations on the OS imposed by Apple
+- Providing root access to the operating system
+- Allowing applications and tools not signed by Apple to be installed and run without any restrictions
+- Debugging and performing dynamic analysis
+- Providing access to the Objective-C Runtime
+
+iOS applications store data in the application sandbox which is not accessible to the public (but is available to root and the application itself). Without root access, it is not possible to access the application sandbox and analyze the data that was stored in it.
+
+#### Caveats and Considerations about Jailbreaking
+
+Jailbreaking iOS devices is becoming more and more complicated as Apple keeps hardening the system, patching the corresponding vulnerabilities that jailbreaks are based on and not signing these vulnerable versions within relative short time intervals (unless they are hardware-based vulnerabilities), meaning that, contrary to Android, you can't downgrade iOS version with one exception explained below. Therefore, jailbreaking has become a very time sensitive procedure. Naturally, this creates a problem, when there is a major bump in iOS version (e.g. from 9 to 10) and there is no public jailbreak for the new OS.
```

Comment:
> Jailbreaking iOS devices ... explained below.

I think this sentence is too long. Furthermore, it contains several subsentences between parenthesis or comma's, which make it hard to read.